c#/Mandoline.Api.Examples/Core/Client/Utilities/SerializableDictionary.cs

```diff
@@ -0,0 +1,208 @@
+﻿namespace SerializeDictionary
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Xml;
+    using System.Xml.Schema;
+    using System.Xml.Serialization;
+
+
+    /// <summary>
+    /// Represents an XML serializable collection of keys and values.
+    /// </summary>
+    /// <typeparam name="TKey">The type of the keys in the dictionary.</typeparam>
+    /// <typeparam name="TValue">The type of the values in the dictionary.</typeparam>
+    [Serializable]
+    [XmlRoot("dictionary")]
+    public class SerializableDictionary2<TKey, TValue> : Dictionary<TKey, TValue>, IXmlSerializable
+    {
+        /// <summary>
+        /// The default XML tag name for an item.
+        /// </summary>
+        private const string DEFAULTITEMTAG = "item";
+
+        /// <summary>
+        /// The default XML tag name for a key.
+        /// </summary>
+        private const string DEFAULTKEYTAG = "key";
+
+        /// <summary>
+        /// The default XML tag name for a value.
+        /// </summary>
+        private const string DEFAULTVALUETAG = "value";
+
+        /// <summary>
+        /// The XML serializer for the key type.
+        /// </summary>
+        private static readonly XmlSerializer keySerializer = new XmlSerializer(typeof(TKey));
+
+        /// <summary>
+        /// The XML serializer for the value type.
+        /// </summary>
+        private static readonly XmlSerializer valueSerializer = new XmlSerializer(typeof(TValue));
+
+        /// <summary>
+        /// Initializes a new instance of the
+        /// <see cref="SerializableDictionary2&lt;TKey, TValue&gt;"/> class.
```

Comment:
Is it correct to have the '&lt' & '&gt' within this "SerializableDictionary2&lt;TKey, TValue&gt;"?

Reply:
I believe this was in the pre-existing client code. Not sure why we are using XML entities rather than the brackets directly.